commercetools-models/src/main/java/io/sphere/sdk/products/commands/updateactions/AddExternalImage.java

```diff
@@ -41,11 +39,11 @@ public static AddExternalImage of(final Image image, final Integer variantId) {
         return new AddExternalImage(image, variantId, null);
     }
 
-    public static AddExternalImage ofVariantId(final Image image, final Integer variantId) {
+    public static AddExternalImage ofVariantId(final Integer variantId, final Image image) {
```

Comment:
yes, changing the signature of `ofVariantId` and `ofSku` is safe since they were not included in the previous release